### PR TITLE
ref: Show warning when user try to use vercel managed org with integration

### PR DESCRIPTION
**Pull request by kamilogorek**
image

image

cc @MildTomato @alaister

### CODE REVIEW

**Review by phamhieu — APPROVED** [review]

File: apps/studio/pages/integrations/vercel/install.tsx [hunk 1]
```diff
@@ -227,11 +231,23 @@ const VercelIntegration: NextPageWithLayout = () => {
                 </AlertDescription_Shadcn_>
               </Alert_Shadcn_>
             )}
+            {hasVercelManagedOrgSelected && (
+              <div className="prose text-sm text-red-900">
```
Comment (joshenlim) [inline apps/studio/pages/integrations/vercel/install.tsx:235]: is the div required? can just apply the class names to <p>

`<p className="prose text-sm text-red-900">`

Comment (kamilogorek) [reply apps/studio/pages/integrations/vercel/install.tsx]: Done

File: apps/studio/pages/integrations/vercel/install.tsx [hunk 2]
```diff
@@ -175,6 +175,10 @@ const VercelIntegration: NextPageWithLayout = () => {
       : false
   }, [installed, selectedOrg, source, dataLoading])
 
+  const hasVercelManagedOrgSelected = useMemo(() => {
+    return !!selectedOrg && selectedOrg.slug.startsWith('vercel_')
```
Comment (alaister) [inline apps/studio/pages/integrations/vercel/install.tsx:179]:
```suggestion
    return !!selectedOrg && selectedOrg.managed_by === 'vercel-marketplace'
```

Comment (kamilogorek) [reply apps/studio/pages/integrations/vercel/install.tsx]: Oh nice, I didnt see that in the request response as it's set in the code itself.

**Review by alaister — APPROVED** [review]
LGTM!

- Tested locally

**Pull request merged**